pkgs/top-level/all-packages.nix

```diff
@@ -5620,6 +5620,7 @@ in
   };
 
   solr = callPackage ../servers/search/solr { };
```

Comment:
Minor nit: I would prefer renaming this with the version as well, so the test can refer explicitly to `solr_7` and then we can have `solr = solr_7;` but other than that I'm very happy to merge!

Reply:
Good point, thanks for mentioning!

Reply:
@srhb all good to merge now?

Reply:
Hey, I pinged you on IRC the other day, but I think I missed you. I changed my mind about a few things: 1. We usally default the _package_ to the newest version and 2. we should probably add a release note because of the stateVersion magic.

So, essentially I suggest this:
https://github.com/srhb/nixpkgs/commit/73fdcd4b147d441e01a9ce6042490839fc28b35b

If you agree, would you mind rolling that into this commit, and perhaps expand the commit message a bit to actually mention the service change?

Sorry that I was slow to reach out :)

Reply:
As discussed on IRC today I totally agree. Thanks for your suggestions and review! I think we're good to go now.